skglm/datafits/single_task.py

```
@@ -230,6 +236,9 @@ def full_grad_sparse(
             grad[j] = yXjyXTw - 1
         return grad
 
+    def intercept_update_step(self, y, Xw):
+        pass
```

would it be better to raise an error instead of ignoring this silently ?